src/logger/isLowerThanOrEqualTo.ts

```diff
@@ -8,7 +8,7 @@ export const isLowerThanOrEqualTo = (
     return false
   }
 
-  if (currentLevel === LogLevel.Debug) {
+  if (currentLevel === LogLevel.Debug || currentLevel === LogLevel.Trace) {
```

Comment:
as `LogLevel.Trace` is higher than `LogLevel.Debug`, I would return `true`from a separate `if` statement